wiki/Guides/BSS_issues/en.md

```diff
@@ -4,35 +4,56 @@ tags:
-5. Extract this `.zip` file.
-6. Open the `.osu` difficulty files with any plain text editor (e.g. Notepad).
-7. Enter anything as username for the `Creator`, but do not use your username.
-8. Set the `BeatmapID` to `0`.
-9. Set the `BeatmapSetID` to `-1`.
-10. Once you are done, zip the files back into the `.zip` file.
-11. Rename this `.zip` file back to `.osz`.
-12. Import it into osu! and try uploading it again.
-13. If this fails, you may want to create a new topic thread in the [Help subforums](https://osu.ppy.sh/community/forums/5) to seek further help.
+You should be able to submit your beatmap by now. However, if you are still having issues, try following these steps:
+
+1. Export the beatmap from inside the beatmap editor by going to `File` and then `Export Package`.
+2. Exit the beatmap editor and delete the beatmap in osu!.
+3. Exit osu!.
+4. Go to the exported beatmap and extract its contents.
+   - Alternatively, open this file directly using external archive utilites such as [WinRAR](https://www.rarlab.com/) or [7-Zip](https://www.7-zip.org/).
+5. Open the `.osu` difficulty files with any text editor (e.g. Notepad).
+6. Fill in the `Creator` field with anything you like, but do **not** use your username.
+7. Set the `BeatmapID` field to `0`.
+8. Set the `BeatmapSetID` field to `-1`.
+9. Once you are done, pack all the files back into one `.osz` file.
+10. Import this file into osu! and try to upload your beatmap.
+
+## Error during upload: Data too long for column "xxx" at row "xx"
+
+This warning occurs when a certain line in the `.osu` difficulty files contains too much text, and can be resolved by following these steps:
+
+1. Go to the beatmap's folder.
+   - You can get there easily by going to `File` and then `Open Song Folder` in the [beatmap editor](/wiki/Client/Beatmap_editor).
+2. Open the `.osu` difficulty files with any text editor. For your convenience, it is recommended that you use an editor that includes line (column) indicators, e.g. [Visual Studio Code](https://code.visualstudio.com/).
+3. Go to the line in the `.osu` difficulty files that's mentioned in the warning.
+4. Shorten the contents of this line by making the necessary adjustments.
+   - If this line concerns metadata matters that can be altered directly through the beatmap editor (e.g. difficulty names), shorten it from there directly through the `Song Setup` panel.
+   - If this line concerns an external asset (e.g. background images), shorten the name of the asset in question and update this line accordingly in your text editor.
+5. Save all the changes you made and try to upload your beatmap.
+
+If the steps above don’t resolve your issue, or if you’re facing a different issue entirely, please create a new topic in the [Help forum](https://osu.ppy.sh/community/forums/5) with details on your issue so that others can help you.
```

Comment:
```suggestion
If the steps above don't resolve your issue, or if you're facing a different issue entirely, please create a new topic in the [Help forum](https://osu.ppy.sh/community/forums/5) with details on your issue so that others can help you.
```